utils/scripts/next_weekly_quiz_generator.py

```diff
@@ -0,0 +1,62 @@
+import json
+import os
+import random
+from datetime import datetime, timedelta
+from firebase_admin import credentials, initialize_app, firestore
+
+
+creds = credentials.Certificate(
+    json.loads(os.environ["FIREBASE_SECRET_KEY_JSON"])
```

Comment:
not related about the refactor. but maybe in the future we could setup some dotenv library so we could try this script easily in local env. I dont mean that it need to be done in this PR, I just comment it here while I remember now :laughing:

Reply:
good idea 👍 
maybe can put it in the [Issues](https://github.com/ia-toki/bebras-pandai/issues) instead of comment, and like any other open source projects raise a PR for the issue is much welcomed ^^